proto/v1alpha1/rode.proto

```diff
@@ -48,6 +50,28 @@ service Rode {
       get: "/v1alpha1/occurrences"
     };
   }
+  rpc CreatePolicy(PolicyEntity) returns (Policy) {
+    option (google.api.http) = {
+      post: "/v1alpha1/policies"
+      body: "*"
+    };
+  }
+  rpc GetPolicy(GetPolicyRequest) returns (Policy) {
+    option (google.api.http) = {
+      get: "/v1alpha1/policies/{id}"
+    };
+  }
+  rpc DeletePolicy(DeletePolicyRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      delete: "/v1alpha1/policies/{id}"
```

Comment:
Will this need to be `{id}/{version}`?

Reply:
Oh yeah.  That's a very good point. I should have that and if they don't provide a version, we assume they want the latest

Reply:
Or maybe we never make assumptions... I'm thinking

Reply:
Resolving this conversation. 
Keeping this as is for now. The version field will remain in the struct, but history will not be maintained.
To be revisited at a later time when we do a POC with a version control system